.storybook/helpers/getArgTypes.ts

```diff
@@ -63,7 +63,11 @@ export function getArgTypes(component, options = { variables: {}}) {
             : value.length < 1
               ? value
               : isNaN(Number(value))
-                ? value
+                ? value === 'null'
+                  ? null
+                  : value.match(/\(\) => \(/gm)
+                    ? new Function(value)()
```

Comment:
Please refactor this entire selection chain. This is 6 tenaries deep, the readability is very poor.